src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,21 @@
 import React from 'react';
+import commentsFromServer from '../../api/comments';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  postId: number,
+};
+
+export const CommentList: React.FC<Props> = ({ postId }) => {
+  const found = (element: Comment) => element.postId === postId;
+
+  return (
+    <div className="CommentList">
+      {commentsFromServer.some(found)
```

Comment:
make util where filter comments from serve by id. and check here length of filtered comments